### PR TITLE
PORISD-146: Update prod domains with archive domains.

### PR DESCRIPTION
**Pull request by tormi**
This PR updates production domains with archive domains following the pori.fi platform switch. Domain changes needs to be provisioned via Ansible.

### CODE REVIEW

File: drupal/.lando/update.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ drush fra -y
 
 # Enable development modules
 drush en -y stage_file_proxy update devel
-drush vset stage_file_proxy_origin 'https://www.pori.fi'
+drush vset stage_file_proxy_origin 'https://pori.prod.wunder.io'
```
Comment (ArtisKrumins) [inline drupal/.lando/update.sh:21]: Site is put under basic auth. Might be this will not work.

File: conf/production.yml [hunk 2]
```diff
@@ -39,9 +39,9 @@
     - name: Set common facts
       set_fact:
         domain1_name: "pori.prod.wunder.io"
-        domain2_name: "pori.fi"
-        domain3_name: "visitpori.fi"
-        domain5_name: "businesspori.fi"
+        domain2_name: "pori.prod.wunder.io"
```
Comment (ArtisKrumins) [inline conf/production.yml:42]: This needs to be provisioned via ansible

File: drupal/.lando/update.sh (hunk 1 above)
Comment (tormi) [reply drupal/.lando/update.sh:21]: Correct. Ho can we bypass basic auth at least for our own VPN?

File: conf/production.yml (hunk 2 above)
Comment (tormi) [reply conf/production.yml:42]: Added related comment to the PR description.

Comment (ArtisKrumins) [reply conf/production.yml:42]: Please also add this under vars

    ```
    basicauth_enabled: True
    basicauth_username: pori
    basicauth_password: kada
    basicauth_ip:
      - address: 84.20.132.177
      - address: 87.94.15.10
      - address: 94.237.27.124
      - address: 94.237.33.88
      - address: 83.136.248.231
      - address: 194.89.239.222
      - address: 194.89.156.118
```

File: drupal/.lando/update.sh (hunk 1 above)
Comment (ArtisKrumins) [reply drupal/.lando/update.sh:21]: Let's add our VPN in config (see below).
So this then should work when using VPN (but not sure actually if Lando/docker takes this into account)

File: conf/production.yml (hunk 2 above)
Comment (tormi) [reply conf/production.yml:42]: [b2a7c66](https://github.com/City-of-Pori/pori-kada/pull/437/commits/b2a7c66f41942e523b46f27ea461a0f5e586cccb)

File: drupal/.lando/update.sh (hunk 1 above)
Comment (tormi) [reply drupal/.lando/update.sh:21]: Done.

**Review by ArtisKrumins — APPROVED** [review]

**Pull request merged**